src/coinselection/greedy.ts

```diff
@@ -32,6 +33,8 @@ function greedyCoinSelection(
   changeAddressGetter: ChangeAddressFromAssetGetter,
   sortFn: CompareUtxoFn
 ): CoinSelectionResult {
+  unspents = unspents.filter(utxo => !isBlindedUtxo(utxo));
```

Comment:
can you explain this?

Reply:
it avoid using blindedUtxos because the algorithm only works with unconfidential unspents

Reply:
shouldn't it work also with conf utxos?

Reply:
well we assume we got them already unblinded before coin selection

Reply:
yeah, I got it now.. maybe better to raise an error here instead of filtering tho? It's mandatory that the unspents provided here have been unblinded eventually.. what do you think?

Reply:
nACK. I think here a blinded utxo mens, no utxos.